app/routes/virtual.$.tsx

```diff
@@ -24,15 +25,9 @@ function handleRedirects(context: LoaderFunctionArgs) {
     {from: '',to: '',},
   ]
 
-  reactVirtualv2List.forEach((item) => {
+  reactVirtualV2List.forEach((item) => {
     if (url.pathname.startsWith(`/virtual/v2/${item.from}`)) {
-      throw redirect(
-        `/virtual/v3/${
-          item.to
-        }?from=reactVirtualV2&original=https://github.com/TanStack/virtual/tree/v2/docs/src/pages/${
-          item.from
-        }${item.from ? '.md' : ''}`
-      )
+      throw redirect(`/virtual/v3/${item.to}?from=reactVirtualV2`)
```

Comment:
This redirect is not thrown for routes that begin with `/virtual/v2/docs` (e.g.: `/virtual/v2/docs/installation`) because those are handled in `app/routes/virtual.$version.docs.$.tsx`.

There's an issue with paths beginning with `/virtual/v2/examples` too, but that wasn't introduced in this PR. Basically, 
```
/virtual/v2/examples/dynamic
``` 
redirects to
```
/virtual/v3/examples/react/dynamic?from=reactVirtualV2
```
instead of
```
/virtual/v3/docs/framework/react/examples/dynamic?from=reactVirtualV2
```

Reply:
Yeah - I might handle this in a separate PR if you're okay with leaving if for now?

Reply:
Yeah, a separate PR sounds fine!